### ISSUE TITLE
"UnquotedVariable" warning should be reporting the offending line in "lineno"

### ISSUE THREAD

**Issue by xgqt** [pkgcore/pkgcheck#442]
Modification of an ebuild file:
```bash
insinto ${S}/asdf
```

Result from the checker:
```bash
/usr/bin/pkgcheck scan --exit\= --reporter\=FlycheckReporter --scopes\=-git --verbose /home/xy/source/public/gentoo.org/repo/gentoo/app-emacs/geiser/geiser-0.26.ebuild
geiser-0.26.ebuild:0:warning:UnquotedVariable: unquoted variable S on line: 31 

Compilation finished at Sat Sep  3 04:31:13
```

![image](https://user-images.githubusercontent.com/47485207/188252491-fc92dfce-8c34-4623-a204-c9e054c833cb.png)

**Comment by arthurzam** [comment]
Just FYI, there are multiple results which collect the lines list in single ebuild to show one result instead of multiple of the same, and `UnquotedVariable` is one of them.

But what I should do is to fix the `FlycheckReporter` so those results which are line collection, split them into multiple results only on that. It will still list in the description all the lines, but you will get multiple reports.

**Comment by xgqt** [comment]
I think listing all offending lines in description is a bad idea - in this case if the cursor would be on offending line (or on hover) other unrelated lines would appear in description.

On September 3, 2022 7:27:47 AM GMT+02:00, Arthur Zamarin ***@***.***> wrote:
>Just FYI, there are multiple results which collect the lines list in single ebuild to show one result instead of multiple of the same, and `UnquotedVariable` is one of them.
>
>But what I should do is to fix the `FlycheckReporter` so those results which are line collection, split them into multiple results only on that. It will still list in the description all the lines, but you will get multiple reports.
>

-- 
Have a great day!
~ Maciej XGQT Barć
